cmd/snap-update-ns/bootstrap.c

```diff
@@ -224,7 +224,7 @@ int validate_snap_name(const char* snap_name)
 }
 
 // process_arguments parses given cmdline which must be list of strings separated with NUL bytes.
-void process_arguments(const char* cmdline, size_t num_read, const char** snap_name_out, bool* should_setns_out)
+void process_arguments(const char* cmdline, const char** snap_name_out, bool* should_setns_out)
```

Comment:
This is fine, but perhaps a comment on how cmdline is guaranteed to be NULL-terminated would be nice.

Reply:
Done